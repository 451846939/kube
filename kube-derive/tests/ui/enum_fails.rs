use kube_derive::CustomResource;
use schemars::JsonSchema;
use serde::{Deserialize, Serialize};

#[derive(CustomResource, Serialize, Deserialize, Debug, Clone, JsonSchema)]
enum FooSpec {
    Foo,
}

fn main() {}
