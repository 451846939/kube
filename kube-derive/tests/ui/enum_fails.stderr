error: Enums or Unions can not #[derive(CustomResource)]
 --> $DIR/enum_fails.rs:6:6
  |
6 | enum FooSpec {
  |      ^^^^^^^
